node/node-sdk.md

```diff
@@ -135,39 +109,38 @@ Workflow code must export a `main` function which will be used as the handler me
 Each workflow instance runs in a separate [V8 isolate][v8-isolate], which means it has a completely isolated execution context including global variables.
 The Temporal SDK injects and removes references from the global object to ensure that code running in the isolate is fully deterministic.
 
-Activities can be referenced by using `Workflow.newActivityStub` and invoked as a method or with options (see below).
+Local activities can be imported and used as normal functions or customized using `Workflow.activity()`.
+- The imported function is replaced with a stub.
+
+Remote activities can be referenced by using `Workflow.activity()`.
 
 ### Example
 ```typescript
+// workflows/src/greeter.ts
+
 import ms from 'ms';
 import { Workflow } from '@temporal-sdk/workflow';
-import { GreeterService } from '@interfaces/activities';
+import { greet } from '@activites/greeter';
+
+// Local activity
+const greetWithCustomTimeout = Workflow.activity(greet, {
+  startToCloseTimeoutMillis: ms('1 hour'),
+});
 
-const greeter = Workflow.newActivityStub<GreeterService>('GreeterService', {
+// Remote activity
+type httpGet = (url: string) => Promise<any>;
+const greetWithCustomTimeout = Workflow.activity<httpGet>('httpGet', {
```

Comment:
I would rename "activity" to newActivityStub and newLocalActivityStub.

Reply:
stub is not a common term in JavaScript world